### ISSUE TITLE
manual/scalaGuide/main/http/ScalaResults.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#1009]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/http/ScalaResults.md
- diffs between [2.2.0/manual/scalaGuide/main/http/ScalaResults.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/main/http/ScalaResults.md) and [2.3.5/manual/scalaGuide/main/http/ScalaResults.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/http/ScalaResults.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/http/ScalaResults.md    2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/main/http/ScalaResults.md   2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Manipulating Results

 ## Changing the default Content-Type
@@ -47,6 +48,10 @@

 @[discarding-cookies](code/ScalaResults.scala)

+You can also set and remove cookies as part of the same response:
+
+@[setting-discarding-cookies](code/ScalaResults.scala)
+
 ## Changing the charset for text based HTTP responses.

 For text based HTTP response it is very important to handle the charset correctly. Play handles that for you and uses `utf-8` by default.

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1062
